dsn_test.go

```diff
@@ -492,6 +544,22 @@ func TestDSN(t *testing.T) {
 	tmfmt := "MM-DD-YYYY"
 
 	testcases := []tcDSN{
+		{
+			cfg: &Config{
+				User:     "u",
+				Password: "p",
+				Account:  "a-aofnadsf.somewhere.azure",
+			},
+			dsn: "u:p@a-aofnadsf.somewhere.azure.snowflakecomputing.com:443?ocspFailOpen=true&region=somewhere.azure&validateDefaultParameters=true",
+		},
+		{
+			cfg: &Config{
+				User:     "u",
+				Password: "p",
+				Account:  "a-aofnadsf.global",
```

Comment:
Shouldn't the account be "aofnadsf" and "a" should be the org?

Reply:
@notkriswagner , do you have any thought?

Reply:
ah nevermind. This is the input, which can include everything before snowflakecomputing.com.

Reply:
And in this case, `a` is the account name and `aofnadsf` will be replication id, because the URL is `global` URL @notkriswagner, can you confirm?

Reply:
Yes thats correct. `a` is the account name, `aofnadsf` is the external ID and this is a global URL so there is no connection to org-name (yet)